### PR TITLE
close tag no line break

### PR DESCRIPTION
**Pull request by chenbihao**
I don't know very well the PR specification of the golang project, please forgive me

### CODE REVIEW

File: xmlfmt_test.go [hunk 1]
```diff
@@ -1,10 +1,8 @@
-package xmlfmt_test
+package xmlfmt
```
Comment (AntonioSun) [inline xmlfmt_test.go]: Please don't make unnecessary changes like this. It was coded like before for a reason.

File: xmlfmt_test.go [hunk 2]
```diff
@@ -18,56 +16,28 @@ const xml4 = `<?xml version="1.0" encoding="UTF-8">
...
-	x3 := xmlfmt.FormatXML(xml3, "\t", " ")
-	x2 := xmlfmt.FormatXML(xml2, "x ", " ")
+	x3 := FormatXML(xml3, "\t", " ")
+	x2 := FormatXML(xml2, "x ", " ")
 	_ = x2
 	_ = x3
-	x1 := xmlfmt.FormatXML(xml1, "", "  ")
+	x1 := FormatXML(xml1, "", "  ")
 	fmt.Println(x1)
-	// Output:
```
Comment (AntonioSun) [inline xmlfmt_test.go:27]: Please don't make unnecessary changes like this. 

When the new change has broken things, hiding it away is not the appropriate way of doing things.

File: xmlfmt_test.go [hunk 3]
```diff
@@ -79,26 +49,21 @@ x  <soapenv:Header/>
 x  <soapenv:Body>
 x   <ns:request>
 x    <ns:customer>
-x     <ns:id>123
-x     </ns:id>
-x     <ns:name type="NCHZ">John Brown
-x     </ns:name>
+x     <ns:id>123</ns:id>
```
Comment (AntonioSun) [inline xmlfmt_test.go:73]: Better add a test case to show what would happen if the string in the middle (between the tags) is super long, more than several lines. What would the output like in that case.

thanks

**Review by AntonioSun — CHANGES_REQUESTED** [review]

File: xmlfmt_test.go (hunk 1 above)
Comment (chenbihao) [reply xmlfmt_test.go]: It has been restored to Package xmlfmt_test , Import "xmlfmt" local package and adds GO.MOD files

File: xmlfmt_test.go (hunk 2 above)
Comment (chenbihao) [reply xmlfmt_test.go:27]: It has been restored

File: xmlfmt_test.go (hunk 3 above)
Comment (chenbihao) [reply xmlfmt_test.go:73]: TestFormatXML_t5：TestFormatXML_t5

**Comment by AntonioSun** [comment]
merged. thanks a lot!
